interface/models.py

```diff
@@ -72,6 +72,7 @@ class Submission(models.Model):
                                    null=True)
     user = models.ForeignKey(User, on_delete=models.PROTECT, null=True)
     output = models.CharField(max_length=4096, default='none')
+    output_err = models.CharField(max_length=4096, default='none')
```

Comment:
What do you say about naming these 2 guys `stdout` and `stderr`? Previously `output` was fine because we concatenated everything in there. Also you will have to do `pipenv run ./manage.py makemigrations` again to update the database changes. To not get another migrations file(one migrations file per PR is what we are aiming for), please delete `interface/migrations/0005_submission_output_err.py`